tutorials/monitoring-demo/index.md

```diff
@@ -1,75 +1,35 @@
 ---
 title: "Monitoring NServiceBus solutions: Demo"
-reviewed: 2017-10-10
+reviewed: 2018-03-15
 summary: A self-contained demo solution that you can run to explore the monitoring features of the Particular Service Platform.
 redirects:
 - tutorials/monitoring/demo
 ---
 
-The best way to get started with the monitoring features in the Particular Service Platform is to try them out with a real system. This downloadable sample contains all of the necessary parts of the platform, already configured and ready to run. It also includes 4 sample endpoints that all communicate by exchanging messages over the SQL Transport.
+The best way to get started with the monitoring features in the Particular Service Platform is to try them out on ServicePulse with a real system. This downloadable sample contains all of the necessary parts of the platform, already configured and ready to run. It also includes 4 sample endpoints that all communicate by exchanging messages over the SQL Transport.
```

Comment:
should "on ServicePulse" be "in ServicePulse"?

Reply:
Think `on`.

@DavidBoike can you confirm?

Reply:
Any of in / on / with would technically work. I prefer on or with.

Reply:
‘With’ would cause repetition. We keep ‘on’.